src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+<style scoped>
+    @import url(/app/pages/details/details.css);
+    @import url(https://fonts.googleapis.com/css?family=Lato:400,900);
+</style>
+<section class="floating-menu">
+    <div class="title">Get this title</div>
+    <div class="get-this-book"></div>
+    <div class="notes">
+        <sup>*</sup>Our web view is Section 508 Level AA accessible.
+    </div>
+    <div class="cta">
+        <p class="interested-text">
+            Using this book, or interested in adopting it for your
+            course?
+        </p>
+        <a href="/adoptions?{{title}}" class="btn">adopt this book</a>
+        <a class="secondary btn">get interested</a>
```

Comment:
If these are intended to be all lowercase, it might be more appropriate to use CSS to `text-transform` it to all lowercase, and code it in the document using more normal English capitalization.